.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,53 @@
+name: Benchmarks Test
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  Benchmarks:
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      matrix:
+        runner: [ubuntu-latest, self-hosted]
```

Comment:
Does this mean that the benchmarks are going to be run on both Github and self-host CI? But we do not need to run on Github CI, whose results are not accurate.